packages/front-end/components/Features/FeatureModal/ProjectSelectField.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from "react";
+import SelectField, {
+  SingleValue,
+  GroupedValue,
+} from "@/components/Forms/SelectField";
+
+const ProjectSelectField: FC<{
+  onChange: (v?: string) => void;
+  value: string;
+  placeholder: string;
+  options: (SingleValue | GroupedValue)[];
+}> = ({ onChange, value, options }) => {
+  return (
+    <SelectField
+      label="Project"
+      value={value}
+      onChange={onChange}
+      placeholder="Select Type..."
+      options={options}
+      required={false}
+      sort={false}
+    />
+  );
+};
+
+export default ProjectSelectField;
```

Comment:
I dont think this component is adding enough value to be it's own component file. I would move this inline to the featureModal or, if there is a way to find commonalities across other project selectors, move it to a common component (outside of the FeatureModal directory).

Reply:
I felt the same TBH. But after going through the description of the issue thought I have to make a separate component. Let me make the changes to keep this inline.